CanonicalLayouts/feed-compose/app/src/main/java/com/example/feedcompose/ui/components/TopAppBar.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.feedcompose.ui.components
+
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.SmallTopAppBar
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import com.example.feedcompose.R
+
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+internal fun TopAppBar(onBackPressed: () -> Unit = {}) {
+    SmallTopAppBar(
+        title = { Text(text = stringResource(id = R.string.app_name)) },
+        navigationIcon = { BackButton(onBackPressed) }
+    )
+}
+
+@Composable
+private fun BackButton(onBackPressed: () -> Unit) {
+    IconButton(onClick = onBackPressed) {
+        Icon(
+            painter = painterResource(id = R.drawable.ic_baseline_arrow_back_24),
+            contentDescription = null
```

Comment:
I know it's sample but we can add values to these to keep accessibility focused.